examples/tests/Earthfile

```diff
@@ -270,6 +270,20 @@ comments-test:
         --mount=type=tmpfs,target=/tmp/earthly \
         -- --no-output +test
 
+empty-line-after-continuation-test:
+    COPY empty-line-after-continuation.earth ./Earthfile
+    RUN --privileged \
+        --entrypoint \
+        --mount=type=tmpfs,target=/tmp/earthly \
+        -- --no-output +test
+
+commented-continuation-line-test:
+    COPY commented-continuation-line.earth ./Earthfile
+    RUN --privileged \
+        --entrypoint \
+        --mount=type=tmpfs,target=/tmp/earthly \
+        -- --no-output +test
```

Comment:
Nice TDD - do both of these pass? It would be good to add these to the ga target. If not, we should add a TODO comment mentioning that it does not pass yet.

Reply:
Im having some problems to run the tests locally, so I was waiting for the pipeline to tell me if they do pass, but I think the pipeline is not calling them by default? Could you point me where should I add change so that they run during the pipeline?

Reply:
To run locally, you can use `./earth -P ./examples/tests+<name-of-target>`.

You would also need to add these to the `ga` target in the same file. Add them to this list: https://github.com/earthly/earthly/blob/master/examples/tests/Earthfile#L12